src/applications/proxy-rewrite/README.md

```diff
@@ -1,80 +1,124 @@
 
-3. **Header may affect font size for the page** to 12px. Specifying font size via scaling may result in too large text loading, then abruptly adjusting to the correct size after the header completes injection.  Other teams have fixed this by resetting the default font size inside the first element which would appear under the header after injection via inline css. This seems to keep the app fonts consistent without the awkward large to small size change on page load.
+## Web components and TeamSites
+Previously, the VA.gov header/footer and the TeamSites header/footer shared quite a bit of code. This helped to reduce tech debt, code duplication and defects.
+
+However, an effort is underway to convert the VA.gov header/footer to web components in the Design System library. Because the code between the VA.gov header/footer and TeamSites header/footer is shared **and** TeamSites do not support web components, we needed to separate the code for Teamsites as much as possible.
+
+As a result, all of the TeamSites header/footer code is now contained in `src/applications/proxy-rewrite` and is 100% duplicative of the VA.gov header/footer. There is a single style file in `proxy-rewrite/sass`, and all of the injection code is still present within this folder. The application has very little dependency on other code within the vets-website repo (with the exception of Platform utilities).
+
+### The downsides of an encapsulated injected header/footer app
+Aside from tech debt and code duplication:
+
+1. Any visual or structural changes made to the VA.gov (modernized) header/footer should also be applied and tested in the injected header/footer to keep the experience consistent
+2. The injected header/footer uses some Platform utility functions. If there are changes to these utilities, you must remember to test those changes against the injected header/footer
+
+## Do I need content-build running locally to test TeamSites?
+**No**. TeamSites do not use `content-build` as they do not use Drupal and thus do not have statically-generated Drupal pages. However, `content-build` generates a static JSON file with all of the Drupal data for the header/footer. You will need a `headerFooter.json` file in `vets-website/build/localhost/generated/`. 
+
+This file is created by running `yarn build` from `content-build`. Without optimization, this build step can take upwards of 8 hours. [Refer to this guide](https://github.com/department-of-veterans-affairs/content-build?tab=readme-ov-file#optimizing-build-time) to optimize your `content-build` and more quickly get up and running. **You will only need to do this once**, or whenever Drupal data for the header/footer is updated and affects your testing. Otherwise, as long as the `headerFooter.json` file exists, you can test the injected header/footer locally.
```

Comment:
Per convo: We need to spell out that you will need to have pulled data recently (which requires SOCKS) in order to have confidence

Reply:
A bit of nuance here I forgot to mention: content-build does not need to be running to load TeamSites locally. In fact, you can't see content-build stuff when you run on the port that TeamSites requires (3001). But you do need to have built content-build recently to get the latest Drupal data for the header/footer. Guidance is updated here.